docs/install/osx_setup.md

```diff
@@ -77,6 +77,10 @@ Install the dependencies, required for MXNet, with the following commands:
 
 ```bash
 	brew update
+    brew install python3
+	brew install cmake
+	brew install ninja
```

Comment:
Same thing for ninja. We can just add a paragraph referring users to ninja and ccache without selling these to them in a combo package with mxnet.

Reply:
I think this is the fastest and most reliable way to build. We have discussed this before.

Reply:
The best install instruction is for users to build mxnet out of box.